india_compliance/gst_india/doctype/gst_settings/gst_settings.json

```diff
@@ -668,12 +669,20 @@
    "label": "Default Reason for e-Invoice Cancellation",
    "mandatory_depends_on": "eval: doc.auto_cancel_e_invoice",
    "options": "Duplicate\nOrder Cancelled\nData Entry Mistake"
+  },
+  {
+   "default": "1",
```

Comment:
default setting should be enabled.
enable this for existing users with a patch.